common/testData/transforms/am-Ethi-t-am-ethi-m0-geminate.txt

```diff
@@ -0,0 +1,6888 @@
+ደናሽ	ደና፟ሽ
+ደን	ደን፟
+ደንቃፋ	ደንቃፋ፟
+ደንበኛ	ደንበኛ፟
+ደንባራ	ደንባራ፟
+ደንባዛ	ደንባዛ፟
+ደንደሮ	ደንደሮ፟
+ደንደሳም	ደንደ፟ሳ፟ም
+ደንደስ	ደንደስ፟
+ደንዳና	ደንዳና፟
+ደንዳኔ	ደንዳኔ፟
+ደንገሎ	ደንገሎ፟
+ደንጉላ	ደንጉላ፟
+ደንጋራ	ደንጋራ፟
+ደንጋዛ	ደንጋዛ፟
+ደከመ	ደከ፟መ
+ደካማ	ደካማ፟
+ደኸየ	ደኸየ፟
+ደወለ	ደወ፟ለ
+ደዋይ	ደዋ፟ይ
+ደደረ	ደደ፟ረ
+ደደቀ	ደደ፟ቀ
+ደደበ	ደደ፟በ
+ደደብ	ደደ፟ብ
+ደጀ	ደጀ፟
+ደጃች	ደጃ፟ች
+ደጃፍ	ደጃ፟ፍ
+ደጅ	ደጅ፟
+ደገመ	ደገ፟መ
+ደገሰ	ደገ፟ሰ
+ደገነ	ደገ፟ነ
+ደገኛ	ደገኛ፟
+ደገደገ	ደገደ፟ገ
+ደገፈ	ደገ፟ፈ
+ደጉሰ	ደጉ፟ሰ
+ደጋደግ	ደጋ፟ደግ፟
+ደጋፊ	ደጋ፟ፊ
+ደግ	ደግ፟
+ደግነቱ	ደግ፟ነ፟ቱ
+ደግነት	ደግ፟ነ፟ት
+ደግዳጋ	ደግዳጋ፟
+ደግፍ	ደግ፟ፍ
+ደጎመ	ደጎ፟መ
+ደጎባ	ደጎ፟ባ
+ደፈረ	ደፈ፟ረ
+ደፈረሰ	ደፈረ፟ሰ
+ደፈቀ	ደፈ፟ቀ
+ደፈነ	ደፈ፟ነ
+ደፈንፋና	ደፈንፋና፟
+ደፈደፈ	ደፈደ፟ፈ
+ደፋ	ደፋ፟
+ደፋር	ደፋ፟ር
+ደፍ	ደፍ፟
+ዱለቻ	ዱለ፟ቻ፟
+ዱለኛ	ዱለ፟ኛ፟
+ዱላ	ዱላ፟
+ዱርዬ	ዱርዬ፟
+ዱባ	ዱባ፟
+ዱቤ	ዱቤ፟
+ዱብ	ዱብ፟
+ዲካ	ዲካ፟
+ዳለቻ	ዳለቻ፟
+ዳለጠ	ዳለ፟ጠ
+ዳሌ	ዳሌ፟
+ዳመነ	ዳመ፟ነ
+ዳመና	ዳመ፟ና
+ዳመናማ	ዳመና፟ማ፟
+ዳመናም	ዳመ፟ናም
+ዳመጠ	ዳመ፟ጠ
+ዳረገ	ዳረ፟ገ
+ዳርቻ	ዳርቻ፟
+ዳርና	ዳርና፟
+ዳሰሰ	ዳሰ፟ሰ
+ዳሸቀ	ዳሸ፟ቀ
+ዳበረ	ዳበ፟ረ
+ዳበሰ	ዳበ፟ሰ
+ዳበሳ	ዳበ፟ሳ
+ዳባ	ዳባ፟
+ዳቦ	ዳቦ፟
+ዳተኛ	ዳተኛ፟
+ዳንሰኛ	ዳንሰኛ፟
+ዳንግሌ	ዳንግሌ፟
+ዳኘ	ዳኘ፟
+ዳኛ	ዳኛ፟
+ዳኝነት	ዳኝ፟ነ፟ት
+ዳከረ	ዳከ፟ረ
+ዳክዬ	ዳክ፟ዬ፟
+ዳዋ	ዳዋ፟
+ዳውላ	ዳውላ፟
+ዳግመኛ	ዳግመኛ፟
+ድህነት	ድህነ፟ት
+ድለላ	ድለ፟ላ
+ድል	ድል፟
+ድልህ	ድል፟ህ
+ድልል	ድል፟ል
+ድልኸ	ድል፟ኸ
+ድልኽ	ድል፟ኽ
+ድልዝ	ድል፟ዝ
+ድልድል	ድልድ፟ል
+ድሎትኛ	ድሎትኛ፟
+ድመራ	ድመ፟ራ
+ድመት	ድመ፟ት
+ድማሪ	ድማ፟ሪ
+ድምር	ድም፟ር
+ድምበረተኛ	ድምበረተኛ፟
+ድምቡሼ	ድምቡሼ፟
+ድሪያ	ድሪ፟ያ
+ድራሹ	ድራ፟ሹ
+ድርቅ	ድር፟ቅ
+ድርቅና	ድርቅና፟
+ድርብ	ድር፟ብ
+ድርት	ድር፟ት
+ድርና	ድርና፟
+ድርድር	ድርድ፟ር
+ድርጁ	ድርጁ፟
+ድርጅት	ድርጅ፟ት
+ድርጊት	ድርጊ፟ት
+ድርግም	ድርግም፟
+ድቁስ	ድቁ፟ስ
+ድቅን	ድቅ፟ን፟
+ድበቃ	ድበ፟ቃ
+ድበብ	ድበብ፟
+ድቡሽት	ድቡ፟ሽት
+ድባኖ	ድባ፟ኖ
+ድብ	ድብ፟
+ድብልቅ	ድብል፟ቅ
+ድብቅ	ድብ፟ቅ
+ድብብቆሽ	ድብብ፟ቆሽ
+ድብት	ድብ፟ት
+ድብን	ድብ፟ን፟
+ድብኝት	ድብኝ፟ት
+ድኅነት	ድኅነ፟ት
+ድንቁርና	ድንቁርና፟
+ድንቄም	ድንቄም፟
+ድንቅ	ድን፟ቅ
+ድንበረተኛ	ድንበረተኛ፟
+ድንበርተኛ	ድንበርተኛ፟
+ድንቡሎ	ድንቡሎ፟
+ድንች	ድን፟ች፟
+ድንክዬ	ድንክዬ፟
+ድንገተኛ	ድንገተኛ፟
+ድንጋጌ	ድንጋ፟ጌ
+ድንጋጤ	ድንጋ፟ጤ
+ድንግልና	ድንግል፟ና
+ድንግዝግዝ	ድንግዝግዝ፟
+ድኝ	ድኝ፟
+ድኩላ	ድኩ፟ላ
+ድኽነት	ድኽነ፟ት
+ድወላ	ድወ፟ላ
+ድው	ድው፟
+ድውር	ድው፟ር
+ድውይ	ድው፟ይ
+ድዳም	ድዳ፟ም
+ድድ	ድድ፟
+ድድር	ድድ፟ር
+ድድቅ	ድድ፟ቅ
+ድገና	ድገ፟ና
+ድጋሚ	ድጋ፟ሚ
+ድግስ	ድግ፟ስ
+ድግን	ድግ፟ን
+ድጎማ	ድጎ፟ማ
+ድጕሳት	ድጕ፟ሳት
+ዶለተ	ዶለ፟ተ
+ዶለዶመ	ዶለዶ፟መ
+ዶልዷላ	ዶልዷላ፟
+ዶልዷሌ	ዶልዷሌ፟
+ዶልዷማ	ዶልዷማ፟
+ዶመኛ	ዶመኛ፟
+ዶኬ	ዶኬ፟
+ዶዮ	ዶዮ፟
+ዶፈዶፈ	ዶፈዶ፟ፈ
+ዶፍዷፋ	ዶፍዷፋ፟
+ጀመረ	ጀመ፟ረ
+ጀምሮ	ጀም፟ሮ
+ጀረገገ	ጀረገ፟ገ
+ጀሶ	ጀሶ፟
+ጀበበ	ጀበ፟በ
+ጀበደ	ጀበ፟ደ
+ጀብደኛ	ጀብደኛ፟
+ጀቦነ	ጀቦ፟ነ
+ጀገነ	ጀገ፟ነ
+ጀግንነት	ጀግንነ፟ት
+ጂጌ	ጂጌ፟
+ጃኬት	ጃኬ፟ት
+ጃዊ	ጃዊ፟
+ጃውሳ	ጃው፟ሳ
+ጃጀ	ጃጀ፟
+ጃፓናዊ	ጃፓ፟ናዊ
+ጃፓን	ጃፓ፟ን
+ጅላንጅል	ጅላንጅ፟ል
+ጅልነት	ጅልነ፟ት
+ጅማታም	ጅማ፟ታም
+ጅማት	ጅማ፟ት
+ጅምር	ጅም፟ር
+ጅብ	ጅብ፟
+ጅንን	ጅን፟ን
+ጅዋጅዌ	ጅዋ፟ጅዌ፟
+ጅው	ጅው፟
+ጆንያ	ጆንያ፟
+ጆፌ	ጆፌ፟
+ጆፍጆፌ	ጆፍጆፌ፟
+ገሀነመ	ገሀነ፟መ
+ገሀነም	ገሀነ፟ም
+ገለል	ገለል፟
+ገለልተኛ	ገለልተኛ፟
+ገለልተኛነት	ገለልተኛ፟ነት
+ገለማ	ገለማ፟
+ገለሞተ	ገለሞ፟ተ
+ገለበ	ገለ፟በ
+ገለበጠ	ገለበ፟ጠ
+ገለጀጀ	ገለጀ፟ጀ
+ገለጠ	ገለ፟ጠ
+ገለጸ	ገለ፟ጸ
+ገለፈጠ	ገለፈ፟ጠ
+ገለፈፈ	ገለፈ፟ፈ
+ገላለጠ	ገላለ፟ጠ
+ገላመጠ	ገላመ፟ጠ
+ገላገለ	ገላገ፟ለ
+ገልዋዳ	ገልዋዳ፟
+ገመለ	ገመ፟ለ
+ገመምተኛ	ገመምተኛ፟
+ገመሰ	ገመ፟ሰ
+ገመተ	ገመ፟ተ
+ገመደ	ገመ፟ደ
+ገመደለ	ገመደ፟ለ
+ገመገመ	ገመገ፟መ
+ገመጠ	ገመ፟ጠ
+ገማ	ገማ፟
+ገማች	ገማ፟ች
+ገምባሌ	ገምባሌ፟
+ገሠሠ	ገሠ፟ሠ
+ገሠጠ	ገሠ፟ጠ
+ገሠጸ	ገሠ፟ጸ
+ገረመ	ገረ፟መ
+ገረመመ	ገረመ፟መ
+ገረረ	ገረ፟ረ
+ገረዘ	ገረ፟ዘ
+ገረደፈ	ገረደ፟ፈ
+ገረጀፈ	ገረጀ፟ፈ
+ገረገረ	ገረገ፟ረ
+ገረገደ	ገረ፟ገደ
+ገረፈ	ገረ፟ፈ
+ገራ	ገራ፟
+ገራም	ገራ፟ም
+ገርበብ	ገርበብ፟
+ገርባባ	ገርባባ፟
+ገርጃፋ	ገርጃፋ፟
+ገርጋራ	ገርጋራ፟
+ገሰረ	ገሰ፟ረ
+ገሰሰ	ገሰ፟ሰ
+ገሰገሰ	ገሰገ፟ሰ
+ገሸለጠ	ገሸለ፟ጠ
+ገሸረ	ገሸ፟ረ
+ገሸሽ	ገሸሽ፟
+ገበረ	ገበ፟ረ
+ገበየ	ገበየ፟
+ገበያተኛ	ገበያተኛ፟
+ገቢና	ገቢና፟
+ገባ	ገባ፟
+ገባር	ገባ፟ር
+ገባች	ገባች፟
+ገባው	ገባ፟ው
+ገብ	ገብ፟
+ገብጋባ	ገብጋ፟ባ
+ገተረ	ገተ፟ረ
+ገተት	ገተ፟ት
+ገታ	ገታ፟
+ገታራ	ገታ፟ራ
+ገትጋታ	ገትጋታ፟
+ገቻ	ገቻ፟
+ገነባ	ገነባ፟
+ገነተረ	ገነተ፟ረ
+ገነት	ገነ፟ት
+ገነነ	ገነ፟ነ
+ገነን	ገነ፟ን
+ገነዘ	ገነ፟ዘ
+ገነደሰ	ገነደ፟ሰ
+ገነገነ	ገነገ፟ነ
+ገነፈለ	ገነፈ፟ለ
+ገና	ገና፟
```

Comment:
ገና has gemination or not depending on meaning, right? is there guidance somewhere about such items?

Reply:
"ገና" is actually _**the**_ classic example where gemination can also change word meaning.  The transliteration files are blind to any context though.  So there is a risk of _unwarranted_ gemination happening, and the user will need to review the output and adjust as needed.  This might be in like ~5% of cases, so overall the gemination is still labor-saving (vs adding gemination where missing currently).  

I can look for words where there is a potential for word meaning to change when geminated and remove words where the geminated form is the less frequent occurrence.